documentation.md

```diff
@@ -38,7 +38,9 @@ HINT: This module can be used with [Mongofill](https://github.com/mongofill/mong
   default: MongoDb::DUMP_TYPE_JS).
 * dump - path to database dump
 * populate: true - should the dump be loaded before test suite is started.
-* cleanup: true - should the dump be reloaded after each test
+* cleanup: true or 'dirty' - should the dump be reloaded after each test.
+  If cleanup is set to 'dirty', the dump is only reloaded if any data has been written to the db during a test. This is
+  checked using the [dbHash](https://docs.mongodb.com/manual/reference/command/dbHash/) command.
```

Comment:
This file is autogenerated from src/Codeception/Module/MongoDb.php 
Please update docblock above class definition.

Reply:
@Naktibalda Done. I also changed the description as I noticed that the value after `:` should actually just be the default value.